frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -69,13 +70,18 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     handlePromise,
     close,
     cancel,
-    allowedBusses = new Set(DiskBus.getAll()),
+    templateValidations,
   } = props;
   const asId = prefixedID.bind(null, 'disk');
   const disk = props.disk || DiskWrapper.EMPTY;
   const volume = props.volume || VolumeWrapper.EMPTY;
   const dataVolume = props.dataVolume || DataVolumeWrapper.EMPTY;
-  const validAllowedBusses = allowedBusses || new Set(DiskBus.getAll());
+  const validAllowedBuses = !templateValidations
```

Comment:
can we do 

```
const tempValidations = templateValidations || new TemplateValidations();
```

before using them? We should get the same default behaviour from the TemplateValidations + it will be defined  just in one place

Reply:
sure, but I didn't like the name though. `temp` prefix is usually referring to `temporary` rather than `template`

Reply:
ye, let your imagination run free :)